definitions/npm/ramda_v0.26.x/flow_v0.104.x-/test_ramda_v0.26.x_relation.js

```diff
@@ -314,3 +315,26 @@ const sym: Array<number> = _.symmetricDifference([1, 2, 3, 4], [7, 6, 5, 4, 3]);
 
 const un: Array<number> = _.union([1, 2, 3])([2, 3, 4]);
 const un1: Array<{ [k: string]: number, ... }> = _.unionWith(eqA, ls1, ls2);
+
+describe('sortBy', () => {
+  it('should take read only in, mutable out', () => {
+    const arr:$ReadOnlyArray<number> = [5, 2, 3, 8];
+    const result1:Array<number> = sortBy(_.identity, arr)
```

Comment:
The reason I use underscore instead of direct import is that I want to establish a pattern that whenever a ramda function is directly imported in current file, it will be used as a main testing subject, where as using underscore to bring in "utilities" to help with testing.

Reply:
Done